py-scripts/DeviceConfig.py

```diff
@@ -81,13 +81,18 @@
 import asyncio
```

Comment:
The commit message on this is very long and becomes truncated partway. There is also no Verified CLI.

Can you please resubmit this patch as two patches, one for CLI help changes and another for skipping passwords with o pen encryption? Please also include the verified CLI. Whitespace/printout changes can go in either this time, but in the future, please submit whitespace/cleanup changes separate from logic changes.

Reply:
Thanks for updating. Commit message is still a bit long, but that's fine for this commit.

In the future, something like 'DeviceConfig.py: skip password for open encryption profile' is shorter and will display better than the existing commit. Current commit is over 80 characters and thus displays like this in GitHub:
```
DeviceConfig.py: Enhanced profile parsing to skip password for open …

…encryption
```